lib/chewy/index.rb

```diff
@@ -240,7 +241,7 @@ def derivable_index_name
       end
 
       # Handling old default_prefix if it is not defined.
-      def method_missing(name, *args, &block) # rubocop:disable Style/MethodMissing
+      def method_missing(name, *args, &block)
```

Comment:
It would be nice to not ignore this check, the better way is to add `respond_to_missing?` method.
Here you can find more info about how to do it http://blog.marc-andre.ca/2010/11/15/methodmissing-politely/ and why it matters